src/plugins/plotting/GuiPlotting.cc

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#include <ignition/plugin/Register.hh>
+#include "GuiPlotting.hh"
+
+using namespace ignition;
+using namespace gui;
+using namespace plugins;
+
+GuiPlotting::~GuiPlotting()
+{
+}
+
+//////////////////////////////////////////
+GuiPlotting::GuiPlotting() : Plugin()
+{
+}
+
```

Comment:
Let's override the `LoadConfig` function so that you can set the title to something like  `Plotting`